tests/elements-regression/tests/elements-regression.test.js

```diff
@@ -41,11 +41,15 @@ test.describe( 'Elementor regression tests with templates for CORE', () => {
 		'sound_cloud',
 		'html',
 		'alert',
+		'button_hover',
```

Comment:
What if we do this?
```suggestion
		{ widgetName: 'button', widgetState: { name: 'hover', selector: 'a', }, },
```
This way, we can be more generic in the code below.

WDYT?

Reply:
I think it will bring more complexity and more code. I will have to refactor the test to use an object instead of just a widget name. We will have two widgets with the same naming and different template files.

Reply:
In in parallel, we are working to add `focus` state to different elements to allow keyboard navigation. Keep this in mind when writing tests.